flac.go

```diff
@@ -48,6 +49,11 @@ type Stream struct {
 	// Zero or more metadata blocks.
 	Blocks []*meta.Block
 	// Underlying io.Reader.
+
+	seekTable *meta.SeekTable
+	// Offset where the music data begins since SeekTable.Offset seems to be relative to this position.
```

Comment:
> since SeekTable.Offset seems to be relative to this position.

We should try to verify this against the official spec and then update the comment accordingly (e.g. remove "seems").

Reply:
I have been testing with a single Flac file that has a seekTable and its first data point is meta.SeekPoint{SampleNum:0, Offset:0}.

Reply:
"Offset (in bytes) from the first byte of the first frame header to the first byte of the target frame's header."

from https://xiph.org/flac/format.html#metadata_block_seektable, so I removed 'seems'